.travis.yml

```diff
@@ -77,6 +77,10 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  #
+  # Some change in virtualenv 14.0.5 caused `test_f2py` to fail. So, we have
+  # pinned `virtualenv` to the last known working version to avoid this failure.
```

Comment:
Yeah, the problem is that AFAICT Travis is more generous than everyone else with respect to how much CPU time they offer to open source projects (e.g. CircleCI gives you 3 parallel workers, Drone.io gives you 1, Travis gives you 6. OTOH I haven't checked how fast Travis's workers are compared to the others...).

I'm 95% sure I saw somewhere where you can download the Travis container images to run locally, but now I can't seem to find it :-(